packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -101,6 +102,8 @@ const AppModals = observer(() => {
 
     const is_eu_user = [ContentFlag.LOW_RISK_CR_EU, ContentFlag.EU_REAL, ContentFlag.EU_DEMO].includes(content_flag);
 
+    const should_show_mt5_notification_modal = mt5_login_list.find(login => login)?.white_label?.notification ?? true;
+
```

Comment:
should this part of code be computed in ui-store and imported like the rest of the modal state?

Reply:
It is like this from start.